frontend/test/metabase/scenarios/native-filters/sql-filters-source.cy.spec.js

```diff
@@ -63,6 +64,10 @@ describe("scenarios > filters > sql filters > values source", () => {
       checkFilterValueNotInList("Gizmo");
       FieldFilter.selectFilterValueFromList("Gadget");
       SQLFilter.runQuery("cardQuery");
+
+      toggleRequired();
```

Comment:
Since text tags and field filters work differently, I've added checks for default values to both of them